### DISCUSSION TITLE
Removing firewall entries

### DISCUSSION THREAD

**Discussion by cyb3rz3us** [Security-Onion-Solutions/securityonion#5472] [Unsupported Versions] [answered]
I recently upgraded from v14 to 2.3 and have noticed that `so-disallow` is no longer available so what is the recommend process to remove stale firewall entries?  I am adept with `iptables` but did not want to modify anything without first checking with the community as to whether or not it is safe to directly manipulate the `iptables`...

**Comment by dougburks** [comment, marked as answer]
Have you checked the firewall documentation?
https://docs.securityonion.net/en/2.3/firewall.html

**Reply by cyb3rz3us** [reply]
As @dougburks  suggested, the key info was found in the **Firewall section of the docs** - specifically the ```so-firewall``` command.
